logstash-input-kafka.gemspec

```diff
@@ -28,7 +28,15 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'logstash-codec-json'
   s.add_runtime_dependency 'logstash-codec-plain'
 
-  s.add_runtime_dependency 'jar-dependencies', ['~> 0.1.0']
+  # Currently there is a blocking issue with the latest (3.1.1.0.9) version of 
+  # `ruby-maven` # and installing jars dependencies. If you are declaring a gem 
+  # in a gemfile # using the :github option it will make the bundle install crash,
+  # before upgrading this gem you need to test the version with any plugins
+  # that require jars.
+  #
+  # Ticket: https://github.com/elasticsearch/logstash/issues/2595
+  s.add_runtime_dependency 'jar-dependencies', '0.1.7'
+  s.add_runtime_dependency 'ruby-maven', '3.1.1.0.8'
   s.add_runtime_dependency 'jruby-kafka', ['>=0.2.1']
```

Comment:
Unrelated to the current PR but this gem should this line somewhere, since it will never work under mri.
```ruby 
s.platform = 'java'
``` 

see https://github.com/logstash-plugins/logstash-output-elasticsearch/pull/56/files#diff-fe5d90447b70d17a85e1b00a598ccee2R37